### DISCUSSION TITLE
dhaqsoPay extension design

### DISCUSSION THREAD

**Discussion by hayle01** [dugsiiyeinc/dhaqsoPay#14] [dhaqsoPay Extension Developers]
waxaan design gareeyay qayb ka mid ah extension-ka waxaan jeclaan lahaa inaad ila aragtaan haddi aan qaadan karno

figma file
https://www.figma.com/design/NYHcgHxMiTQpBw1w2nsN2h/dhaqsoPay?node-id=0-1&t=x79gfbo101aBpMBt-1

![image](https://github.com/user-attachments/assets/83963941-519d-4f37-9728-f6b1814656bb)

**Comment by sharafdin** [comment]
> waxaan design gareeyay qayb ka mid ah extension-ka waxaan jeclaan lahaa inaad ila aragtaan haddi aan qaadan karno
> 
> figma file https://www.figma.com/design/NYHcgHxMiTQpBw1w2nsN2h/dhaqsoPay?node-id=0-1&t=x79gfbo101aBpMBt-1
> 
> ![image](https://private-user-images.githubusercontent.com/136359097/386245518-83963941-519d-4f37-9728-f6b1814656bb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE1OTczMzksIm5iZiI6MTczMTU5NzAzOSwicGF0aCI6Ii8xMzYzNTkwOTcvMzg2MjQ1NTE4LTgzOTYzOTQxLTUxOWQtNGYzNy05NzI4LWY2YjE4MTQ2NTZiYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExNFQxNTEwMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYjRlNjFjMWI2YjJiOGI2NGQ4MjJkZTI5MDNkMTJmOWY2ZWU5ZTZjYTA0MzVmZDMyYWM1YjgxMjBiMTk3MGViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Smfswq5iU5SM_Enz0m_oM_lrLWbteETMVWTib6QP8To)

aniga ahaan waa 80% done. lakin dark mode ka page ka cadiga wali waa cadaan ama light kaliya dashboard ka ayaa is bdalay.

**Comment by sharafdin** [comment]
legenow deisgns kaaga waa fadhiyaa.

**Comment by sharafdin** [comment]
mida kale 252 inee is badasho mahan oo wax is bedelaayo inuu noqdo mahan. sido kale based on company-ga uu doortay waa in loo sii diyaariyaa prefix kiisa sida, hormuud -> 61, somtel -> 62

**Reply by hayle01** [reply]
Taasi waan la socdaa 252 selection ma ahan. marka uu doorto company ayaa loo diyaarinaya prefix. iyo tan dark mode-ka ah labadaba waan diyaarinaya haddi aan isku raacnay. sidoo kale meesha banaan wax la  dhigo haddi aad haydo nala wadaag eng

**Reply by sharafdin** [reply]
> Taasi waan la socdaa 252 selection ma ahan. marka uu doorto company ayaa loo diyaarinaya prefix. iyo tan dark mode-ka ah labadaba waan diyaarinaya haddi aan isku raacnay. sidoo kale meesha banaan wax la dhigo haddi aad haydo nala wadaag eng

kwrm hadii aan latest transctions dhigno meesha banaan ? design kaan waa cool light iyo dark theme isagoo ah soo habee waayo hadeer dark theme ka xitaa ma toosna sido kalena kuso dar latest transactions. colors, logo, icons waad iga heysaa. dhanka font-gana adeegsada roboto

**Comment by iftiiin** [comment]
M.a design-ka wuu diyaarsanyhy
